src/validators/index.ts

```diff
@@ -42,7 +42,48 @@ export const ArticleValidator = new Validator(
   },
 );
 
+export const ArticleUuidValidator = new Validator({
```

Comment:
You could reuse, `uuid` validator

Reply:
I think we still need the ArticleUuidValidator with pagination fields add on, don't we?

Reply:
We need a method to reuse schema or concat schema.